src/functions-reference/higher-order_functions.Rmd

```diff
@@ -141,6 +142,131 @@ Stan's automatic differentiation.
 
 ## Ordinary Differential Equation (ODE) Solvers {#functions-ode-solver}
 
+Stan provides several higher order functions for solving initial value problems specified as Ordinary Differential Equations (ODEs).
+
+Solving an initial value ODE means given a set of differential equations $y' = f(y, t)$ and initial conditions $y(t_0)$, solving for $y$ at a sequence of times $t_0 < t_1 \leq t_2, \cdots \leq t_n$. $f(y, t)$ is referred to here as the ODE system function.
```

Comment:
Here we should make the parameters $\theta$ explicit in f and y, I think... or say that we suppress them on purpose.